lib/octokit/client/issues.rb

```diff
@@ -329,6 +329,19 @@ def issue_timeline(repo, number, options = {})
       def add_assignees(repo, number, assignees, options = {})
         post "#{Repository.path repo}/issues/#{number}/assignees", options.merge({:assignees => assignees})
       end
+
+      # Remove assignees from an issue
+      #
+      # @param repo [Integer, String, Repository, Hash] A GitHub repository
+      # @param number [Integer] Issue number
+      # @param assignees [Array] Assignees to be removed
+      # @return [Sawyer::Resource] Issue
+      # @see https://developer.github.com/v3/issues/assignees/#remove-assignees-from-an-issue
+      # @example Remove assignees "pengwynn" and "joeyw" from Issue #23 on octokit/octokit.rb
+      #   Octokit.remove_assignees("octokit/octokit.rb", 23, ["pengwynn", "joeyw"])
+      def remove_assignees(repo, number, assignees)
```

Comment:
This is missing the normal patter of an `options` hash as the last argument, would you mind adding it?

```ruby
def remove_assignees(repo, number, assignees, options = {})
  # ...
end
```

Reply:
Sounds good. Just updated the commit.